Sources/Nimble/Matchers/BeAKindOf.swift

```diff
@@ -58,9 +58,9 @@ public func beAKindOf(_ expectedClass: AnyClass) -> Predicate<NSObject> {
 }
 
 extension NMBObjCMatcher {
-    @objc public class func beAKindOfMatcher(_ expected: AnyClass) -> NMBMatcher {
-        return NMBObjCMatcher(canMatchNil: false) { actualExpression, failureMessage in
-            return try! beAKindOf(expected).matches(actualExpression, failureMessage: failureMessage)
+    @objc public class func beAKindOfMatcher(_ expected: AnyClass) -> NMBPredicate {
+        return NMBPredicate { actualExpression in
+            return try! beAKindOf(expected).satisfies(actualExpression).toObjectiveC()
```

Comment:
Curious, can this closure be throwing? Would be nice to replace `try!` with `try` (although I'm not familiar with the possible failures here).

Reply:
Let's revisit that in a separate PR (targeting to `master` branch).